tests/test_study.py

```diff
@@ -684,6 +684,55 @@ def test_append_trial(storage_mode):
         assert study.best_value == 0.8
 
 
+@pytest.mark.parametrize('storage_mode', STORAGE_MODES)
+def test_enqueue_trial_properly_sets_param_values(storage_mode):
+    # type: (str) -> None
+
+    with StorageSupplier(storage_mode) as storage:
+        study = optuna.create_study(storage=storage)
+        assert len(study.trials) == 0
+
+        study.enqueue_trial(params={'x': -5, 'y': 5})
+        study.enqueue_trial(params={'x': -1, 'y': 0})
+
+        def objective(trial):
+            # type: (optuna.trial.Trial) -> float
```

Comment:
Could you add a blank line after this comment, per convention?
```suggestion
            # type: (optuna.trial.Trial) -> float
            
```

Reply:
These changes make the CI fail because autopep8 removed a blank line. Can I revert it? One solution is to use a function annotation syntax.

Reply:
Ah, this is because whitespaces are included. You can either remove the whitespaces, or use function annotations. Either is fine. Sorry for my mistake.

Reply:
Oh, I see. I'll remove white spaces. Thanks.